packages/boot/src/booters/base-artifact.booter.ts

```diff
@@ -48,7 +52,7 @@ export class BaseArtifactBooter implements Booter {
   /**
    * `glob` pattern to match artifact paths
    */
-  glob: string;
+  globs: string[];
```

Comment:
This looks like a breaking change to me, can you please confirm?

Should we preserve backwards compatibility e.g. by implementing `glob` getter/setter that will use `globs[0]` entry as the storage?